### PR TITLE
Adding "--provider hyperv" to "vagrant up" in README files for HyperV

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

**Comment by praveenkumar** [comment]
LGTM

**Comment by naina-verma** [comment]
Error when --provider hyperv is missing ,
https://paste.fedoraproject.org/378981/59111971/

File: components/rhel/misc/hyperv/rhel-k8s-singlenode-hyperv/README.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ QuickStart
 
 4. Start the Vagrant box.
 
-  ``$ vagrant up``
+  ``$ vagrant up --provider hyeprv``
```
Comment (praveenkumar) [inline components/rhel/misc/hyperv/rhel-k8s-singlenode-hyperv/README.rst]: it should be `hyperv` looks like `hyeprv` a typo.

**Comment by praveenkumar** [comment]
@LalatenduMohanty Can you also update commit message it still have typo.

**Comment by LalatenduMohanty** [comment]
@praveenkumar done.

**Comment by praveenkumar** [comment]
LGTM

**Pull request merged**